.editorconfig

```diff
@@ -0,0 +1,21 @@
+# This is the project top-level .editorconfig
+root = true
+
+# Defaults for all file types
+[*]
+# 4-space indents, no TABs
+indent_style = space
+tab_width = 4
+indent_size = tab
+
+# Hard-wrap at 120 columns
+max_line_length = 119
+
+# Unix EOL
+end_of_line = crlf
```

Comment:
Unix EOL should be lf, so either the comment needs changing or the config (preferably for me, the config), or just remove it entirely and let git handle it?

Reply:
Changed the comment, and I even resisted saying 'hysterical' rather than 'historical'.